src/middlewares/user.validation.js

```diff
@@ -0,0 +1,13 @@
+import db from '../models';
+import Responses from '../utils/response';
+
+// eslint-disable-next-line require-jsdoc
+const findByEmail = async (req, res, next) => {
```

Comment:
Good job here Mastel! I feel we could have here, a name that is more **reflective** of what this function is doing?

Reply:
Also add JSDoc here. There's probably an extension in your IDE that could help with that.